control-plane/csi-driver/src/bin/node/main_.rs

```diff
@@ -144,6 +144,13 @@ pub(super) async fn main() -> anyhow::Result<()> {
                 .value_parser(clap::value_parser!(bool))
                 .help("Enable ansi color for logs")
         )
+        .arg(
+            Arg::new("stale-entry-cleanup")
+                .long("stale-entry-cleanup")
+                .action(clap::ArgAction::SetTrue)
+                .value_name("BOOLEAN")
+                .help("Enable cleanup of the stale entries before controller publish")
```

Comment:
why do we need to enable this on the node? Shouldn't it be a controller feature?
Should it be a flag considering it could lead to corruption if not used?

Reply:
Okay yeah maybe we should do on controller. Okay maybe a flag with SetFalse as default?

Reply:
Done.

Reply:
Removing for now.